homeassistant/components/amcrest/camera.py

```diff
@@ -365,10 +370,18 @@ def update(self):
                     self._brand = "unknown"
             if self._model is None:
                 resp = self._api.device_type.strip()
+                _LOGGER.debug("Device_type=%s", resp)
                 if resp.startswith("type="):
                     self._model = resp.split("=")[-1]
                 else:
                     self._model = "unknown"
+            if self._serial is None:
+                resp = self._api.serial_number.strip()
+                _LOGGER.debug("serial=%s", resp)
+                if resp.startswith("type="):
```

Comment:
What is the amcrest version and camera you are using and what is the response you are seeing here? The current `amcrest` package version that is being used (1.8.0) should already handle parsing the value here to return just the serial number.

Reply:
Basically, with the current package, I don't think this logic and splitting should be needed from the `self._api.serial_number` output.

Reply:
You are correct. I copied the type= from the code above, but it should be removed. My debug log line is 
```
2021-08-13 11:14:32 DEBUG (SyncWorker_3) [custom_components.amcrest.camera] serial=AMC0XXXXXXXXX
```